apps/wallet/ios/Podfile.lock

```diff
@@ -993,4 +999,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 877abdf091993fa0cd980670b68c31cf31b2d28e
 
-COCOAPODS: 1.14.3
```

Comment:
I have no idea which version of pods that we should use, please double check this one